instrumentation/netty/netty-4.1/javaagent/src/test/groovy/Netty41ClientTest.groovy

```diff
@@ -287,21 +285,6 @@ class Netty41ClientTest extends HttpClientTest implements AgentTestTrait {
     null != channel.pipeline().remove('Netty41ClientTest$OtherSimpleHandler#0')
   }
 
-  def "when a traced handler is added from an initializer we still detect it and add our channel handlers"() {
```

Comment:
This test genuinely fails on  Netty 4.1.0.Final, and only works starting from 4.1.5.Final,  see https://github.com/netty/netty/pull/5588

It worked before this PR only because test used AHC, which used netty version newer than 4.1.0.

Reply:
Indeed, will do that